src/huggingface_hub/keras_mixin.py

```diff
@@ -100,6 +102,7 @@ def _create_model_card(
     model,
     repo_dir: Path,
     plot_model: Optional[bool] = True,
+    tags: Optional[Union[list, str]] = None,
     task_name: Optional[str] = None,
```

Comment:
Since this is a private method (as denoted by the `_`), I think you can freely remove `task_name` from here